app/connector/email/pom.xml

```diff
@@ -99,6 +99,10 @@
       <artifactId>spotbugs-annotations</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Is this only for the `com.google.common.net.HttpHeaders`?

I'd avoid adding a dependency on Guava just for that. Guava is difficult dependency to depend on as it evolves rapidly and (sometimes) breaks backward compatibility. We might end up with multiple versions of Guava simply because of different project life-cycles and this can cause compatibility issues.

Reply:
lol!
I just added it in because maven dependencies analysis complained it was missing. Seems it depended on by commons-util. None of my changes actually actively use it.